replicate-to-sql-with-generic-events/scripts/delete-data.js

```diff
@@ -0,0 +1,35 @@
+const { Kuzzle, WebSocket } = require('kuzzle-sdk');
+
+const kuzzle = new Kuzzle(new WebSocket('localhost'));
+
+const indexName = 'nyc-open-data';
+const collectionName = 'yellow-taxi';
+
+async function deleteData(ids = []) {
+  try {
+    const result = kuzzle.document.mDelete(indexName, collectionName, ids);
+    await kuzzle.collection.refresh(indexName, collectionName);
+    return result;
+  }
+  catch (error) {
+    throw error;
+  }
+}
+
+async function run() {
+  try {
+    await kuzzle.connect();
+    const result = await kuzzle.document.search(indexName, collectionName, {}, { size: 500 });
+    const willDeleteIds = result.hits.map(d => d._id);
+    const response = await deleteData(willDeleteIds);
+    console.log(`Deleted ${response.successes.length} documents`);
+  }
+  catch (error) {
+    throw error;
```

Comment:
Redundant with how exceptions work, this catch block should be removed.

(you can still have a finally block without a catch one)